hepdata/utils/miscellaneous.py

```diff
@@ -20,9 +20,52 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+import bleach
+import html
+
 def splitter(data, predicate):
     """Split a list according to a given predicate (lambda)."""
     yes, no = [], []
     for d in data:
         (yes if predicate(d) else no).append(d)
     return yes, no
+
+
+def sanitize_html(value, tags=None, attributes=None, strip=False):
+    """Sanitize HTML.
+
+    :param tags: Allowed HTML ``tags``. Configuration set by Invenio-Config.
+    :param attributes: Allowed HTML ``attributes``. Configuration set by
+        Invenio-Config.
+    :param strip: Whether to strip tags that are not allowed. Defaults to
+        False (escapes rather than strips disallowed tags).
+
+    Use this function when you need to include unescaped HTML that contains
+    user provided data.
+    """
+    from flask import current_app
+
+    # The lines below were copied from invenio-formatter but with the addition
+    # of the strip parameter.
+    value = value.strip()
```

Comment:
The use of `strip` gives an exception if `value` is `None`, for example, for a newly created record without a data abstract.  I guess you just need to add something like:
```python
if value is None:
    return value
```